setup.py

```diff
@@ -1,9 +1,20 @@
 from skbuild import setup
 
+version = '{major}.{minor}.{patch}'
+
+with open('VERSION.cmake', 'r+') as version_file:
+    version_text = version_file.read()
+    # split on lines, remove empty lines
+    lines = filter(len, version_text.split('\n'))
+    # parse out digit characters from the line, convert to int
+    numbers = [int("".join(filter(str.isdigit, line))) for line in lines]
```

Comment:
If it would be a pure python package, I would suggest to use [python-versioneer](https://github.com/warner/python-versioneer)

But in the case of `createrepo_c`, the project can be built either as a standalone project or python package, the version management needs to be available to both.

For now, I suggest to stick with your approach. Later after we standardize approaches  like the one done [here](https://github.com/SimpleITK/SimpleITK/blob/master/CMake/sitkSourceVersion.cmake), this could be revisited.